packages/circuit-ui/util/test-utils.tsx

```diff
@@ -22,6 +22,7 @@ import {
   waitFor,
   act,
   RenderResult,
+  fireEvent,
```

Comment:
🤔 , should we not only keep `userEvent` to encourage its use? Or does `fireEvent` do anything that `userEvent` doesn't?

Reply:
`userEvent` doesn't support the `popState` event, but `fireEvent` does. That's why I had to add it.